orchestrator/index.js

```diff
@@ -0,0 +1,67 @@
+const request = require('request-promise');
+const HABANERO = require('node-red-habanero');
+
+const HABANERO_ADMIN_ROOT = '/orchestrator';
+const HABANERO_NODE_ROOT  = '/orchestrator-api';
+
+function init(server, app){
+	// Habanero
+	var habaneroSettings = require('../node_modules/node-red-habanero/settings');
```

Comment:
Use const and move the `require` to the top of the file. Avoid using inline `require` calls.